### PR TITLE
validator: add role to install

### PR DESCRIPTION
**Pull request by fromanirh**
Add support to deploy the template validator, using
the https://github.com/openshift/service-serving-cert-signer
facility provided by OKD/OCP

Replaces https://github.com/MarSik/kubevirt-ssp-operator/pull/4

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
Needs to be tested. On it.

File: build/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,11 @@
 FROM quay.io/water-hole/ansible-operator
 
+USER 0
+COPY repos/RPM-GPG-KEY-CentOS-SIG-PaaS /etc/pki/rpm-gpg
+COPY repos/centos-openshift-origin.repo /etc/yum.repos.d
+RUN yum install -y openssl origin-clients
+USER 1001
```
Comment (fromanirh) [inline build/Dockerfile]: Uhm, we don't actually need this (and related repo files), cleaning up

**Comment by fromanirh** [comment]
seems to work. I want to do another verification e2e with updated common-templates, but we can merge this PR

File: roles/KubevirtTemplateValidator/files/service.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  template:
+    metadata:
+      name: virt-template-validator
+      labels:
+        kubevirt.io: virt-template-validator
+    spec:
+      serviceAccountName: template-validator
+      containers:
+        - name: webhook
+          image: quay.io/fromani/kubevirt-template-validator:v0.3.0
```
Comment (MarSik) [inline roles/KubevirtTemplateValidator/files/service.yaml]: Can we extract this to a variable (role default)? Just so we can make downstream builds easier.

**Pull request merged**